packages/manager/modules/account-migration/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@ovh-ux/manager-account-migration",
+  "version": "0.0.0",
+  "private": true,
+  "description": "OVHcloud Migration product",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/account-migration"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
```

Comment:
There are build scripts but we never use the dist. We should change this entry to target the dist and also add a `module` entry. Not sure about the browser entry (umd) though

Reply:
This is done. Thanks :)